scripts/create_lightning_rc.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+OLDVER=0.37.0
+LVER=0.38.0
```

Comment:
Not for now, of course, but I'm super curious if it would be too complicated to query the version from the version file and set these values automatically.